app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -46,6 +47,13 @@ Not qualified as a teacher yet? Find out [how to train to teach in England as a
 
 If you do not want to come to England to train, you should consider [international qualified teacher status (iQTS)](/non-uk-teachers/international-qualified-teacher-status), which leads to the automatic award of English qualified teacher status. 
 
+<section class="inset-text purple">
```

Comment:
This looks fine on the front-end but I thought we normally defined these insets in the frontmatter and then called them in the body?

Paging @emmaachesongray for advice!